azure-pipelines.yml

```diff
@@ -0,0 +1,101 @@
+# Author: Sammy Pfeiffer <Sammy.Pfeiffer at student.uts.edu.au>
+jobs:
+  - job: "build_sbre_kinetic_os_1"
+    timeoutInMinutes: 0
+
+    pool:
+      vmImage: 'Ubuntu 16.04'
+
+    variables:
+      docker_image_name: sbre_os_kinetic_32b
+
+    steps:
+    - script: cd ros_base; docker build --network host -f Dockerfile.from_roogp -t $(DockerHub_user)/$(docker_image_name):$(Build.BuildId) .
+      displayName: 'docker build'
+      # This means up to 6h
+      timeoutInMinutes: 0
+    - script: docker login -u $(DockerHub_user) -p $(DockerHub_password)
```

Comment:
You'd need to add as Variables in Azure Pipelines these variables `DockerHub_user` and, as a secret one, `DockerHub_password`.